.gitignore

```diff
@@ -16,6 +16,9 @@ packages/contracts/coverage*
 packages/contracts/@ens*
 packages/contracts/@openzeppelin*
 packages/contracts/hardhat*
+packages/contracts/artifacts-ovm*
+packages/contracts/cache-ovm*
```

Comment:
These no longer exist and you should just clean the folder structure to remove them